content/cli/workspaces.md

```diff
@@ -190,6 +191,76 @@ These properties specify the compiler to use as well as various options that aff
 | `deleteOutDir`      | boolean             | If `true`, whenever the compiler is invoked, it will first remove the compilation output directory (as configured in `tsconfig.json`, where the default is `./dist`).                                                                    |
 | `assets`            | array               | Enables automatically distributing non-TypeScript assets whenever a compilation step begins (asset distribution does **not** happen on incremental compiles in `--watch` mode).  See below for details.                                  |
 
+#### Global generate options
+
+These properties specify the default generate options for when running the `nest generate` command.
```

Comment:
Making a few suggestions.  The following is the first, and is the format I will use.

"for when running" -> "to be used by"

Above means I suggest replacing existing text (left of arrow) with the suggestion to the right of the arrow.